468067  - "pkg-config --libs OpenIPMIpthread" fails

Fix the pkgconfig file.

Reported upstream as http://sourceforge.net/tracker/index.php?func=detail&aid=2188750&group_id=36127&atid=416301

diff -up OpenIPMI-2.0.18/OpenIPMIpthread.pc.in.orig OpenIPMI-2.0.18/OpenIPMIpthread.pc.in
--- OpenIPMI-2.0.18/OpenIPMIpthread.pc.in.orig	2005-08-17 05:56:02.000000000 +0200
+++ OpenIPMI-2.0.18/OpenIPMIpthread.pc.in	2010-05-20 15:48:07.000000000 +0200
@@ -6,6 +6,6 @@ includedir=@includedir@
 Name: OpenIPMIpthread
 Description: Pthread OS handler for OpenIPMI
 Version: @VERSION@
-Requires: OpenIPMI pthread
+Requires: OpenIPMI
 Libs: -L${libdir} -lOpenIPMIutils -lOpenIPMIpthread
-Cflags: -I${includedir}
+Cflags: -I${includedir} -pthread
